src/main/java/slimeknights/tconstruct/library/smeltery/IFaucetDepth.java

```diff
@@ -0,0 +1,10 @@
+package slimeknights.tconstruct.library.smeltery;
+
+/**
+ * Block interface to determine how far the seared faucet liquid flows into a block
+ */
+public interface IFaucetDepth {
```

Comment:
Why did you add this? We removed it already for the faucet. I want a proper model solution.

Reply:
hm. Don't remember putting that in. Perhaps when I was splitting the faucet commits
out I left that behind